frontend/lib/tools/auto.ts

```diff
@@ -30,6 +31,21 @@ export async function autoAnswer(
         const newMessages = getHistoryMessages(isPro, messages);
         const query = newMessages[newMessages.length - 1].content;
 
+        let summaryTitle = '';
+        let summaryText = '';
+
+        try {
+            const { text } = await generateText({
+                model: getLLM('gpt-4o-mini'),
+                system: 'You are a professional writer. You write simple, clear, and concise content.',
+                prompt: `Summarize the following article in 3-5 sentences: ${query}`,
```

Comment:
Please give the following question a concise title

Reply:
Got it!